include/tvm/json.h

```diff
@@ -0,0 +1,317 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
let us keep the base64 and json in the src. The main reason is that we do not want to expose these APIs as public API, but rather internal ones. Likely keeping these files in the original location won't affect anything in the parser. As we only use them via save/load json